trek.js

```diff
@@ -0,0 +1,131 @@
+$(document).ready( function() {
+
+  let loadTrips = function loadTrips(){
+    $('#load').hide();
+    // $('#trips').show();
+    const urlTripName = 'https://trektravel.herokuapp.com/trips';
+    $.get(urlTripName,
+      (response) => {
+        $('h1').html('List of Trips');
+        for (trip of response) {
+          let tripName = `<li><a data-id=${trip.id}> ${trip.name}</a>
+          </li>`
+
+          $('#trips ul').append(tripName);
+        }
+      }).fail((response) => {
+        let badRequest = `<p class=bad >Something went wrong loading trip options</p>`
+        $('#ajax-results').html(badRequest);
+        console.log("Didn't go so hot");
+      });
+    };
+
+    // End loadTrips
+
+    let loadOneTrip = function loadOneTrip(tripId){
+      $('.good').hide();
+      let urlTripDetails = 'https://trektravel.herokuapp.com/trips/'
+      urlTripDetails += tripId;
+      $.get(urlTripDetails,
+        function(response) {
+          let details = `
+          <div class="large-8 columns">
+          <h2> ${response.name}</h2>
+          <table>
```

Comment:
Good job wrapping this dynamic content in a sectioning element. Much cleaner than having a bunch of free-floating `<p>` tags.